### PR TITLE
Update nl translation

### PR DESCRIPTION
**Pull request by eirvandelden**
Thanks for your interest in plotly.js!

### Translations:

- People who can assist in verifying the dutch words: @rriksma-fouriq, @shendriks-fouriq, @ariejan
- Keys I've committed are buttons we don't use in our own application and I'm not confident enough to translate without any context. Especially `hover` would translate to a very verbose way

### Features, Bug fixes, and others:

Before opening a pull request, developer should:

- [x] 1. make sure they are not on the `master` branch of their fork as using `master` for a pull request would make it difficult to fetch `upstream` changes.

- [x] 2. fetch latest changes from `upstream/master` into your fork i.e. `origin/master` then pull `origin/master` from you local `master`.

- [x] 3. then `git rebase master` their local dev branch off the latest `master` which should be sync with `upstream/master` at this time.

- [x] 4. make sure to **not** `git add` the `dist/` folder (the `dist/` is updated only on version bumps).

- [x] 5. make sure to commit changes to the `package-lock.json` file (if any new dependency required).

- [x] 6. provide a title and write an overview of what the PR attempts to do with a link to the issue they are trying to address.

- [x] 7. select the _Allow edits from maintainers_ option (see this [article](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) for more details).

After opening a pull request, developer:
 

- [x] - should create a new small markdown log file using the PR number e.g. `1010_fix.md` or `1010_add.md` inside `draftlogs` folder as described in this [README](https://github.com/plotly/plotly.js/blob/master/draftlogs/README.md), commit it and push.

 - should **not** force push (i.e. `git push -f`) to remote branches associated with opened pull requests. Force pushes make it hard for maintainers to keep track of updates. Therefore, if required, please fetch `upstream/master` and "merge" with master instead of "rebase".

### CODE REVIEW

**Comment by eirvandelden** [comment]
I've also added the draft log file, though I'm not sure if it was required for a translation change. If anything else is needed, let me know!

File: lib/locales/nl.js [hunk 1]
```diff
@@ -3,7 +3,41 @@
...
+      'Click to enter Component B title': 'Klik om Component B titel in te vullen',
+      'Click to enter Component C title': 'Klik om Component C titel in te vullen',
+      'Click to enter Plot title': 'Klik om Plot titel in te vullen',
+      'Click to enter X axis title': 'Klik om x-as titel in te vullen',
+      'Click to enter Y axis title': 'Klik om y-as titel in te vullen',
+      'Click to enter radial axis title': 'Klik om radiaal-as titel in te vullen',
+      'Double-click to zoom back out': 'Zoom uit door te dubbel klikken',
+      'Download plot as a png': 'Dowload de plot als een png-bestand',
+      'Download plot': 'Download de plot',
+      'Draw circle'; 'Teken cirkel',
```
Comment (archmoj) [inline lib/locales/nl.js:19]:
```suggestion
      'Draw circle': 'Teken cirkel',
```

Comment (eirvandelden) [reply lib/locales/nl.js]: Woops, sorry, I fixed the typo 🙈

File: lib/locales/nl.js [hunk 2]
```diff
@@ -18,6 +52,12 @@ module.exports = {
             'jan', 'feb', 'maa', 'apr', 'mei', 'jun',
             'jul', 'aug', 'sep', 'okt', 'nov', 'dec'
         ],
-        date: '%d-%m-%Y'
+        date: '%d-%m-%Y',
+        decimal: ',',
+        thousands: '.'
```
Comment (archmoj) [inline lib/locales/nl.js:57]:
```suggestion
        thousands: '.',
```

File: lib/locales/nl.js [hunk 3]
```diff
@@ -3,7 +3,41 @@
...
+      'Download plot as a png': 'Dowload de plot als een png-bestand',
+      'Download plot': 'Download de plot',
+      'Draw circle': 'Teken cirkel',
+      'Draw closed freeform': 'Teken gesloten vorm',
+      'Draw line': 'Teken lijn',
+      'Draw open freeform': 'Teken open vorm',
+      'Draw rectangle': 'Teken rechthoek',
+      'Edit in Chart Studio': 'In Chart Studio wijzigen',
+      'Erase active shape': 'Wis huidige vorm',
+      'IE only supports svg. Changing format to svg.': 'IE ondersteund alleen svg bestanden. Formaat gewijzigd naar svg.',
```
Comment (rriksma-fouriq) [inline lib/locales/nl.js:26]:
```suggestion
      'IE only supports svg. Changing format to svg.': 'IE ondersteunt alleen svg bestanden. Formaat gewijzigd naar svg.',
```

**Comment by archmoj** [comment]
Please add spaces here so that the test pass & and we could possibly include it in the next plotly.js minor soon.
Thank you!

```sh
$ eslint .
```

```
/home/circleci/plotly.js/lib/locales/nl.js
   7:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
   8:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
   9:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  10:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  11:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  12:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  13:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  14:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  15:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  16:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  17:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  18:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  19:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  20:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  21:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  22:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  23:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  24:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  25:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  26:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  27:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  28:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  29:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  30:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  31:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  32:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  33:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  34:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  35:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  36:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  37:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  38:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
  39:7  error  Expected indentation of 8 spaces but found 6  indent-legacy
```

**Comment by archmoj** [comment]
:dancer:
I'll fix the linting right after.
The updates would be in `v2.14.0` shortly.
Thanks very much.

**Comment by eirvandelden** [comment]
> Please add spaces here so that the test pass & and we could possibly include it in the next plotly.js minor soon. Thank you!
> 

Sorry that I didn't apply thee added spaces, I was on holiday🙈. Thanks for changing, glad to have helped!

**Pull request merged**